docs/calico-with-docker/AWS.md

```diff
@@ -323,20 +329,20 @@ docker run -v /usr/local/bin:/target jpetazzo/nsenter
 ```
 
 #### Modify container Calico interface MTU
-For each container on your host, enter the namespace of the container, 
+For each container on your host, enter the namespace of the container,
 replacing `<container_id>` with the name or ID of the container.
 ```
 sudo nsenter -n -t $(sudo docker inspect --format '{{ .State.Pid }}' <container_id>) /bin/bash
 ```
 
-Finally, modify the MTU of the Calico interface on the container. If you 
-specified an interface when adding the container to Calico, change `eth1` to be 
+Finally, modify the MTU of the Calico interface on the container. If you
+specified an interface when adding the container to Calico, change `eth1` to be
 the name of the interface you passed in.
 ```
-sudo ifconfig eth1 mtu 8981
+sudo ifconfig eth1 mtu 9001
```

Comment:
Ah, I see, I think I misread the above. I think this is fine as 9001, but good to note that here should be 8981 also if using ip-ip.